src/views/TradingCompetition/components/PrizesInfo/PrizesGrid/index.tsx

```diff
@@ -17,7 +17,7 @@ import {
   TrophyGoldIcon,
 } from '@pancakeswap/uikit'
 import { useTranslation } from 'contexts/Localization'
-import easterPrizes, { Tiers, Rank } from 'config/constants/trading-competition/easter'
+import easterPrizes, { Tiers, Rank } from 'config/constants/trading-competition/prizes'
```

Comment:
maybe lets rename easter stuff at the same time to avoid confusion 🙂

Reply:
👍